src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -563,6 +565,25 @@ public function nodeName()
      * @throws \InvalidArgumentException When current node is empty
      */
     public function text(/* $default = null */)
```

Comment:
`$default` is not used anymore? I fear that we have a conflict here between `$default` and the new normalize whitespace argument.

Reply:
I will check that

Reply:
we use func_get_arg(0)

Reply:
I can see that, but still, there is a conflict here.

Reply:
Have a look at the `textContent()` method. It looks for the default value but it is not passed. That does not work.

Reply:
That was not working properly know it's working !